templates/base-site.conf

```diff
@@ -5,7 +5,18 @@ server {
     server_name {{ site.urls | join(" ") }};
 
     {% if site.tls is defined %}
+
+    {% if nginx_acme_challenge_path is defined %}
+    location /.well-known/acme-challenge/ {
+        alias {{ nginx_acme_challenge_path }};
+    }
+    location / {
+        return 301 https://{{ site.urls | first }}$request_uri;
+    }
+    {% else %}
     return 301 https://{{ site.urls | first }}$request_uri;
```

Comment:
I changed this a bit, to only add a `location /` if `nginx_acme_challenge_path` is defined.
If its not defined it will result in the same configuration like it was before the commit with just a return without any location blocks.